Sources/HermesAppDelegate.m

```diff
@@ -69,13 +69,9 @@ - (NSMenu *)applicationDockMenu:(NSApplication *)sender {
   NSMenuItem *menuItem;
   Song *song = [[playback playing] playingSong];
   if (song != nil) {
-    [menu addItemWithTitle:@"Now Playing" action:nil keyEquivalent:@""];;
-    [menu addItemWithTitle:[NSString stringWithFormat:@"   %@", [song title]]
-                    action:nil
-             keyEquivalent:@""];
-    [menu addItemWithTitle:[NSString stringWithFormat:@"   %@", [song artist]]
-                    action:nil
-             keyEquivalent:@""];
+    [menu addItemWithTitle:nowPlaying.title action:nil keyEquivalent:@""];;
```

Comment:
Should mention I do like this change though — I had thought the window title would always be there to show the station name but of course that is not always the case.